Jenkinsfile

```diff
@@ -15,7 +15,18 @@ node {
         ])
     }
 
-    stage('build') {
-        sh "docker-compose build"
-    }
+    stage('Build Docker') {
```

Comment:
You can call these like this now:
https://github.com/plangrid/plangrid-build-tools/blob/dev/vars/StandardBuild.groovy#L41